### PR TITLE
Test | `getHousehold query happy & sad path testing`

### LINKED ISSUE #20: Test | `Happy & Sad Path Testing for getHousehold query`
- [x] Happy path test the mutation

- [x] Sad path test the mutation

### PR DESCRIPTION
**Pull request by hamouj**
# Description
This PR will:
-Create get_household_spec with happy and sad path testing for getHousehold query
-Update household factory to use email faker (vs unsafe email which is deprecating soon)
-Add filter to SimpleCov to ignore graphql_controller and to_do_guru_api_schema generated by graphql-ruby gem
<br>

## Type of Changes:

- [ ] Bug fix
- [ ] New feature
- [ ] Refactoring
- [x] Testing

<br>
<br>

# Related Issue(s):

- [x]  closes #20  

<br>

## Other Considerations 

- [x] I have linted my code locally before submission
- [x] This change requires a documentation update (add error response to docs)

### CODE REVIEW

File: spec/graphql/queries/get_household_spec.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      expect(result.dig("data", "household", "sunday", 0, "assignedMember")).to eq(member1.name)
+      expect(result.dig("data", "household", "sunday", 0, "duration")).to eq(chore1.duration)
+      expect(result.dig("data", "household", "monday")).to be nil
+      expect(result.dig("data", "household", "tuesday")).to be nil
+      expect(result.dig("data", "household", "wednesday", 0, "choreName")).to eq(chore2.name)
+      expect(result.dig("data", "household", "wednesday", 0, "assignedMember")).to eq(member2.name)
+      expect(result.dig("data", "household", "wednesday", 0, "duration")).to eq(chore2.duration)
+      expect(result.dig("data", "household", "thursday")).to be nil
+      expect(result.dig("data", "household", "friday")).to be nil
+      expect(result.dig("data", "household", "saturday")).to be nil
```
Comment (aj-bailey) [inline spec/graphql/queries/get_household_spec.rb:40]: I dig this

Comment (aj-bailey) [inline spec/graphql/queries/get_household_spec.rb:40]: I dig this

File: spec/factories/households.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :household do
     name { "The #{Faker::Name.last_name}'s House" }
-    email { Faker::Internet.safe_email }
+    email { Faker::Internet.email }
```
Comment (aj-bailey) [inline spec/factories/households.rb:4]: Thanks for updating this

**Review by aj-bailey — APPROVED** [review]
Looks great, thanks for updating the simplecov and factorybot

**Review by MoxieMax — APPROVED** [review]

**Review by Bphayes1200 — APPROVED** [review]
Looks good!

**Pull request merged**